index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en-us">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Go Green - Summit 2022</title>
+    <meta name="description" content="Go Green - Summit 2022">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="stylesheets/styles.css">
+</head>
+
+<body>
+    <section class="top-section">
+        <div class="top-cover">
+            <nav class="inactive">
+                <a href="#close"><img src="images/bars.png" alt="Nav Button"></a>
+                <a href="index.html" class="logo"><img src="images/green_logo.png" alt="logo"></a>
+                <ul>
+                    <li class="Home">
+                        <a href="#">Home</a>
+                    </li>
+
+                    <li>
+                        <a href="about-page.html" class="active">About</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">Program</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">Join</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">Sponser</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">News</a>
+                    </li>
+                    
+                    <li>
+                        <a href="#">CC Compaign</a>
+                    </li>
```

Comment:
- [ ] Hi, Please notice that this link in the reference design has a colored border around it. since you are required to match the design and colors please implement this so it fits closer to the reference design. 

|Your Design |Template Design |
-|-
|![image](https://user-images.githubusercontent.com/18730106/152666204-134965a3-b159-4ecb-9803-170cd9e281b4.png)|![image](https://user-images.githubusercontent.com/18730106/152666201-0d6e4ad5-42d2-4033-977c-430e513a8f66.png)|